src/main/java/sirius/biz/translations/BasicTranslations.java

```diff
@@ -58,11 +74,23 @@ protected BasicTranslations(BaseEntity<?> owner) {
     /**
      * Provides a validator so translations can only be added for supported languages.
      *
-     * @param lang the language code in question
+     * @param lang  the language code in question
+     * @param field the name of the field to be translated
      * @return true if language is supported by the system, false otherwise
+     * @throws sirius.kernel.health.HandledException if lang is not supported by the system
      */
-    protected boolean isSupportedLanguage(String lang) {
-        return supportedLanguages.contains(lang);
+    protected boolean isValidLanguage(String lang, String field) {
+        boolean isSupported = validLanguages.isEmpty() || validLanguages.contains(lang);
+
+        if (!isSupported) {
```

Comment:
this is very unexpected behavior - isXXX returning true/false should never throw instead of returning false.

Rename to "assert / ensureValidLanguage" and remove return type?